mailpoet/lib/Newsletter/Scheduler/PostNotificationScheduler.php

```diff
@@ -162,7 +163,8 @@ public function processPostNotificationSchedule(NewsletterEntity $newsletter) {
     $intervalType = $intervalTypeOption ? $intervalTypeOption->getValue() : null;
 
     $timeOfDayOption = $newsletter->getOption(NewsletterOptionFieldEntity::NAME_TIME_OF_DAY);
-    $hour = $timeOfDayOption ? (int)$timeOfDayOption->getValue() / self::SECONDS_IN_HOUR : null;
+    $hour = $timeOfDayOption ? (int)floor((int)$timeOfDayOption->getValue() / self::SECONDS_IN_HOUR) : null;
+    $minute = $timeOfDayOption && $hour ? ((int)$timeOfDayOption->getValue() - (int)($hour * self::SECONDS_IN_HOUR)) / self::SECONDS_IN_MINUTE : null;
```

Comment:
Will this code also work for the time between midnight and 1 am (e.g., 00:45)? If I understand the code correctly in such case, the `$hour` will be 0, so `$timeOfDayOption && $hour ` will evaluate as `false` and `$minute` will be `null`. Maybe we don't need the `&& $hour` here.

Reply:
Also with the `null` value in `$minute` and `$hour = 0` the  `sprintf('%s %s * * *', $minute, $hour);` will evaluate to `' 0 * * *'` which seems to be an incorrect value for cron timing. I tried checking in at `https://crontab.guru/#_0_*_*_*`

Reply:
Great catch 👍 Thank you